contracts/mapping-token/contracts/interfaces/IMessageVerifier.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.10;
+
+interface IMessageVerifier {
+    function bridgedChainPosition() external view returns(uint32);
+    function bridgedLanePosition() external view returns(uint32);
+    function thisChainPosition() external view returns(uint32);
+    function thisLanePosition() external view returns(uint32);
+}
```

Comment:
@hujw77 Please help review this interface.

Reply:
The interfaces are OK, but I need to review the use case later.